github/ci/prow-deploy/kustom/overlays/ibmcloud-production/resources/bootstrap.yaml

```
@@ -1,4 +1,4 @@
-apiVersion: v1
+piVersion: v1
```

This is an error, right?
```suggestion
apiVersion: v1
```

---

Absolutely, this file is not currently included but as soon as we add it we would hit an error, thanks!